### PR TITLE
added ubuntu platform support

### PR DESCRIPTION
**Pull request by vkhatri**

### CODE REVIEW

**Comment by jmauro** [comment]
Hey,

Thanks for the Pull request but there is a dependency issue, since you are using 'apt_repository' apt as to be a dependency could you add it?


Regards,
JM

**Comment by vkhatri** [comment]
@jmauro thank you! i missed it.

pushed a commit for it. please let me know if need to re base and submit another PR.

File: attributes/repo.rb [hunk 1]
```diff
@@ -4,3 +4,9 @@
 default['rundeck_server']['yum']['baseurl']     = 'http://dl.bintray.com/rundeck/rundeck-rpm/'
 default['rundeck_server']['yum']['gpgkey']      = 'http://rundeck.org/keys/BUILD-GPG-KEY-Rundeck.org.key'
 default['rundeck_server']['yum']['action']      = :create
+
+default['rundeck_server']['apt']['uri'] = 'http://dl.bintray.com/rundeck/rundeck-deb/'
+default['rundeck_server']['apt']['components'] = %w(/)
+default['rundeck_server']['apt']['key'] = 'http://rundeck.org/keys/BUILD-GPG-KEY-Rundeck.org.key'
+default['rundeck_server']['apt']['options'] = '--force-yes'
```
Comment (rveznaver) [inline attributes/repo.rb]: Why are you forcing yes here?

**Comment by rveznaver** [comment]
Please remove the ```manage realm.properties configuration file``` commit as it is already in another pull request.

File: attributes/repo.rb (hunk 1 above)
Comment (vkhatri) [inline attributes/repo.rb]: `apt-get` was reporting issue when using `-y install` which is default options for `apt_package` and adding `--force-yes` was a proposed solution.

However, package can be installed if i execute `apt-get install rundeck` on shell without any issue. To be honest, i did not have time to investigate it further.

I will give it another try, if there is no further issue with the PR. Will remove the extra commit.

Appreciate your feedback.

**Comment by rveznaver** [comment]
Hi @vkhatri, your other commit has been merged, so you can rebase this one.

**Pull request closed without merging**